### PR TITLE
Fix pages:build on MacOS

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

Like the title says, this change fixed the `pages:build` script on MacOS systems.

## Instructions for local reproduction and review

Check if it still works for you on Windows with `npm run pages:build && npm run pages:build:serve`

### CODE REVIEW

File: scripts/buildPages.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@ for i in "${array[@]}"
 do
   echo "Building $i docs ..."
 	npm run docs:$i
-  mkdir --parents ./pages/docs/$i
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    mkdir -p ./pages/docs/$i
+  else
+    mkdir --parents ./pages/docs/$i
+  fi
```
Comment (warm-coolguy) [inline scripts/buildPages.sh:10]:
```suggestion
  mkdir -p ./pages/docs/$i
```
We can drop the OS check, `-p` also works on git bash on Windows.

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
🏓 @dopenguin

**Comment by dopenguin** [comment]
> 🏓 @dopenguin

🏓 @warm-coolguy

**Review by warm-coolguy — APPROVED** [review]
🥇 @dopenguin

**Pull request merged**